src/core/sdk/store.ts

```diff
@@ -10,7 +10,76 @@ import {
 } from "../../types";
 import { checkIsTestnet } from "./testnet";
 import { persist, createJSONStorage } from "zustand/middleware";
-import { get, set, del } from 'idb-keyval'; // You'll need to install this package
+
+// Determine if we're in a browser environment
+const isBrowser = typeof window !== 'undefined' && window.indexedDB;
+
+// Create appropriate storage
+const getStorage = () => {
+  if (isBrowser) {
+    // Browser storage using IndexedDB
+    const { get, set, del } = require('idb-keyval');
+    return {
+      getItem: async (name: string) => {
+        return (await get(name)) || null;
+      },
+      setItem: async (name: string, value: any) => {
+        await set(name, value);
+      },
+      removeItem: async (name: string) => {
+        await del(name);
+      },
+    };
+  } else {
+    // Node.js storage using filesystem
+    // Use dynamic imports to avoid bundling Node.js modules in browser builds
+    try {
+      // This code will only run in Node.js environments
+      const fs = require('fs');
```

Comment:
Has this been tested as part of a build/import to check that its still working in a node environment?